APP_DEVELOPMENT.md

```diff
@@ -84,6 +85,46 @@ In case you need it, you can also use `useColorMode` or `useColorModeValue` to d
 
 **To find which colours to use for dark mode in Figma** see *Design System* in the list of Pages on the left-hand side of the screen.  There are examples of some components using dark mode.
 
+### Custom components
```

Comment:
Suggest that we keep this updated as a kind of easy gallery of custom components.  Obviously we can look at the code files too, but this is a fast, visual reference.